app/src/main/java/space/taran/arknavigator/mvp/model/repo/preview/PreviewStorage.kt

```diff
@@ -10,5 +10,5 @@ interface PreviewStorage {
 
     fun forget(id: ResourceId)
 
-    fun generate(path: Path, meta: ResourceMeta)
+    fun store(path: Path, meta: ResourceMeta)
 }
```

Comment:
I don't think it needs to be renamed.

Reply:
NP, what about this comment: https://github.com/ARK-Builders/ARK-Navigator/pull/313#discussion_r998221579
@kirillt

Reply:
@mdrlzy yeah, not really an issue of course, but IMO `generate` sounds like we generate a data, but here we only store existing data by path (we generate a file, but not the data).